Externals/wxWidgets3/CMakeLists.txt

```diff
@@ -818,23 +818,20 @@ set(SRCS_UNIXGTK
 	"src/unix/uiactionx11.cpp"
 	"src/unix/utilsx11.cpp")
 
-include_directories(.)
-include_directories(include)
-
-set(SRCS
-	${SRCS_AUI}
-	${SRCS_COMMON}
-	${SRCS_GENERIC})
+add_library(wx STATIC ${SRCS_AUI} ${SRCS_COMMON} ${SRCS_GENERIC})
```

Comment:
Should this be explicitly static? AFAIK, builds are static by default, and this would prevent using `BUILD_SHARED_LIBS` or w/e with it (assuming anyone wanted to).